build/yamls/base/controller-rbac.yml

```diff
@@ -112,6 +113,14 @@ rules:
       - patch
       - create
       - delete
+  - apiGroups:
```

Comment:
maybe remove this block for now.. since we don't yet have code in for ee

Reply:
Done. Will add back for externalEntity support PR